services/discord.js

```diff
@@ -111,9 +132,11 @@ exports.buildActiveStreaksMessage = () => {
   }).join('\n')
 }
 
-exports.assignActiveStreakRole = userID => {
-  const user = client.guilds.get(constants.DevStreakGuildID).members.find(u => u.id === userID)
-  user.addRole(constants.ActiveStreakerRoleID, 'Has a top streak at the end of the day')
+exports.assignActiveStreakRole = (guild, userID) => {
```

Comment:
Why pass in the entire guild here? Why not just the guildID?

Reply:
because we need to find the user, which was done by looking through the members of the guild and finding the user with the given id. `const user = guild.members.find(u => u.id === userID)`, however this could also be done by passing the user instead of the userID, either way you either need to pass the user or the guild.